resources/public/api/conf/5.0/brought_forward_losses_retrieve.yaml

```diff
@@ -1,15 +1,24 @@
 summary: Retrieve a Brought Forward Loss
 description: |
-  This endpoint allows the developer to retrieve an existing brought forward loss amount.
+  
+  This endpoint enables you to retrieve an existing brought forward loss amount. 
```

Comment:
Please revert this for now as it is inconsistent with other endpoint descriptions.